server/config/subject_page.proto

```diff
@@ -279,6 +279,10 @@ message Tile {
   // [optional] If set, used instead of main place for the page.
   string place_dcid_override = 11;
 
+  // [optional] used only in ranking tiles
+  // set to true to hide the footer with download / explore more link
+  bool hide_footer = 17;
```

Comment:
nit: maybe should put this in RankingTileSpec since it's specific to ranking tiles?

Reply:
Looked into this, if we put `hide_footer` into `RankingTileSpec` in the proto, cleanest implementation is to also move `hideFooter` into `rankingMetadata` on the tile side, to avoid type errors in `block.tsx` but that feels a little odd to me. Thoughts?

Reply:
Going to merge this for now, and we can followup with a props cleanup PR after some more discussion about how to refactor/abstract.